crates/floresta-electrum/src/electrum_protocol.rs

```diff
@@ -335,7 +341,7 @@ impl<Blockchain: BlockchainInterface> ElectrumServer<Blockchain> {
                 );
                 json_rpc_res!(request, res)
             }
-            // TODO: Return peers?
```

Comment:
NIT: This method return addresses of other Electrum Servers, so it would be `TODO: Return servers?`. But given that floresta is a trusted personal server, I don't see why your wallet would need to find new nodes, so it's ok to remove this TODO all along.